framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -299,6 +299,29 @@ void MediaPlayerImpl::unpreparePlayer(player_result_t &ret)
 	return notifySync();
 }
 
+player_result_t MediaPlayerImpl::reset()
+{
+	LOG_STATE_INFO(mCurState);
+	meddbg("MediaPlayer reset mPlayer : %x\n", &mPlayer);
+	
+	if (mCurState == PLAYER_STATE_READY || mCurState == PLAYER_STATE_PLAYING || mCurState == PLAYER_STATE_PAUSED) {
+		mInputHandler.close();
+		if (reset_audio_stream_out(mStreamInfo->id) != AUDIO_MANAGER_SUCCESS) {
+			meddbg("MediaPlayer reset fail : reset_audio_stream_out fail\n");
+			return PLAYER_ERROR_INTERNAL_OPERATION_FAILED;
```

Comment:
In my opinion, if another thread already use audio manager, it should keep going on release resource in media.
so I think some warning message is enough, we should not return error.

Reply:
Thanks.
Changes Done as requested.